usaspending_api/api_contracts/contracts/v2/reporting/agencies/agency_code/publish_dates.md

```diff
@@ -0,0 +1,79 @@
+FORMAT: 1A
```

Comment:
@jonhill13 Just realized the API responses for this endpoint and `/api/v2/reporting/agencies/publish_dates/history/` are flipped. The publish history is the history for a single submission. The filters and sorting aren't applicable for the history response.